extra-network/smartdns/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=smartdns
+PKGSEC=net
+PKGDES="A local DNS server to obtain website IP with fastest connection response"
```

Comment:
A local DNS server which resolves domain IP addresses based on the fastest response